external/avro/src/main/scala/org/apache/spark/sql/v2/avro/AvroScan.scala

```diff
@@ -65,4 +65,6 @@ case class AvroScan(
   }
 
   override def hashCode(): Int = super.hashCode()
+
+  override def getMetaData(): Map[String, String] = super.metaData
```

Comment:
We need this? It seems `FileScan` already has the implementation.

Reply:
@maropu I get a compile error that forces me to implement it here ? Let me know if you have any suggestion.

Reply:
I changed the code in `FileScan` and the compilation passed;
```
-  protected def getMetadata(): Map[String, String] = {
+  override def getMetaData(): Map[String, String] = {
```

Reply:
@maropu Thanks a lot :-) I will make a change.